components/card/ProfilePersonCard.vue

```diff
@@ -224,6 +242,11 @@ export default {
       &--linkedin {
         background-color: var(--color-linkedin-01);
       }
+      &--askformentorship {
```

Comment:
color ve font-weight property'sini kesinlikle sil. `background-color: var(--color-ui-05);`olarak tanımla veya color.css'de social altında `--color-ask-for-mentorship-01` olarak yeni bi color key oluşturup onu kullan